src/workers/notifications/squadMemberJoined.ts

```diff
@@ -24,6 +31,14 @@ const worker: NotificationWorker = {
         role: SourceMemberRoles.Admin,
       },
     });
+
+    const actionType =
```

Comment:
This is a bit more tricky than that, because now admins can promote other admins.
And these new admins didn't really create a squad

Reply:
Yes, though it was discussed to provide the checklist for owners even if it came from a promotion.

Sab mentioned:

> We need to understand that this means if a user becomes a Squad admin (owner) then they see the owner checklist and it replaces any older checklists that may or may not have been completed

https://dailydotdev.slack.com/archives/C0506KU4RH9/p1681985308241999?thread_ts=1681978406.223319&cid=C0506KU4RH9

Another relevant discussion:

https://dailydotdev.slack.com/archives/C0506KU4RH9/p1683019582216549

Reply:
Ok!